package.json

```diff
@@ -47,19 +55,19 @@
     "url": "https://github.com/18F/analytics-reporter/issues"
   },
   "dependencies": {
-    "googleapis": "^2.1.5",
-    "node-schedule": "^0.1.13",
-    "minimist": "*",
     "async": "*",
     "aws-sdk": "*",
+    "cfenv": "^1.0.3",
```

Comment:
We should make `cfenv` and `newrelic` optional dependencies, for those who don't use them.